platform/linux-generic/odp_schedule_basic.c

```diff
@@ -60,14 +64,10 @@ ODP_STATIC_ASSERT((ODP_SCHED_PRIO_NORMAL > 0) &&
 /* Not a valid index */
 #define NULL_INDEX ((uint32_t)-1)
 
-/* Priority queue ring size. In worst case, all event queues are scheduled
- * queues and have the same priority. The ring size must be larger than or
- * equal to ODP_CONFIG_QUEUES / QUEUES_PER_PRIO, so that it can hold all
- * queues in the worst case. */
-#define PRIO_QUEUE_RING_SIZE  (ODP_CONFIG_QUEUES / QUEUES_PER_PRIO)
-
-/* Mask for wrapping around priority queue index */
-#define RING_MASK  (PRIO_QUEUE_RING_SIZE - 1)
+/* Maximum priority queue ring size. A ring must be large enough to store all
+ * queues in the worst case (all queues are scheduled, have the same priority
+ * and no spreading). */
+#define MAX_RING_SIZE ODP_CONFIG_QUEUES
```

Comment:
Do we require `ODP_CONFIG_QUEUES` to be a power of two? If not should this be rounded up here?

Reply:
Ring size needs to be power of 2. It's optimal for memory consumption and performance that ring size is not overly large. So, it's optimal to keep these two defines in sync.